components/navigable-container/index.js

```diff
@@ -124,19 +124,28 @@ export class NavigableMenu extends Component {
 		const eventToOffset = ( evt ) => {
 			const { keyCode } = evt;
 
-			const isVertical = orientation === 'vertical';
-			const isHorizontal = orientation === 'horizontal';
-
-			// Still handle any arrow keys, even if the opposite orientation
-			if ( LEFT === keyCode ) {
-				return isHorizontal ? -1 : 0;
-			} else if ( UP === keyCode ) {
-				return isVertical ? -1 : 0;
-			} else if ( RIGHT === keyCode ) {
-				return isHorizontal ? +1 : 0;
-			} else if ( DOWN === keyCode ) {
-				return isVertical ? +1 : 0;
-			}
+			const offsetMap = {
+				vertical: {
+					[ LEFT ]: 0,
+					[ RIGHT ]: 0,
+					[ UP ]: -1,
+					[ DOWN ]: +1,
+				},
+				horizontal: {
+					[ LEFT ]: -1,
+					[ RIGHT ]: +1,
+					[ UP ]: 0,
+					[ DOWN ]: 0,
+				},
+				vertical_two_columns: {
```

Comment:
Do we really want to hard-code the number of columns into an `orientation` value?

Reply:
Also if the list is something like that:

* *
*
* *

It doesn't work, I don't think we should implement this "smart" behavior, I think we should provide a wa to have horizontal + vertical do the same thing "+1/-1". I think it's an ok tradeoff